src/vs/workbench/contrib/interactiveEditor/browser/interactiveEditorController.ts

```diff
@@ -811,32 +808,24 @@ function installSlashCommandSupport(accessor: ServicesAccessor, editor: IActiveC
 
 async function showMessageResponse(accessor: ServicesAccessor, query: string, response: string) {
 	const interactiveSessionService = accessor.get(IInteractiveSessionService);
-	const providerId = interactiveSessionService.getProviders()[0];
-	if (await interactiveSessionService.revealSessionForProvider(providerId)) {
-		interactiveSessionService.addCompleteRequest(providerId, query, { message: response });
+	const providerId = interactiveSessionService.getProviderIds()[0];
```

Comment:
FYI @jrieken, changes in this file, everything should be working the same though